Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -1381,6 +1381,28 @@ There are several ways to perform dynamic analysis:
 1. For the actual persistence: Use the techniques described in the data storage chapter.
 2. For reflection-based approaches: Use Xposed to hook into the deserialization methods or add unprocessable information to the serialized objects to see how they are handled (e.g., whether the application crashes or extra information can be extracted by enriching the objects).
 
+### Testing for Tapjacking Vulnerability (MSTG-PLATFORM-9)
+
+#### Overview
+
+Tapjacking is a vulnerability wherein the malicious application leverages screen overlay feature of android to capture user taps, keystrokes. That is because an active screen overlay can listen for taps and intercept any information being passed in the underlying activity.
```

Comment:
```suggestion
Tapjacking is a vulnerability in which the malicious application abuses the screen overlay feature of Android to capture user taps and keystrokes. This is possible because an active screen overlay can listen for taps and intercept any information being passed in the underlying activity.
```

Reply:
Do you have any examples of real world apps that were abusing this? Would be good to mention here, links to blogs, etc.

Reply:
Good idea, it'd help a lot